app/src/sharedTest/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -283,95 +295,107 @@ class RevisionCardFragmentTest {
+      onView(withId(R.id.previous_navigation_card)).check(matches(not((isDisplayed()))))
     }
   }
 
+//  TODO(#4631): Remove this once #4235 is resolved.
+  @DisableAccessibilityChecks
   @Test
   fun testRevisionCardTestActivity_fractionSubtopicId1_checkPreviousAndNextNavCardsDisplayed() {
     launch<RevisionCardActivity>(
       createRevisionCardActivityIntent(
         context,
         profileId.internalId,
         FRACTIONS_TOPIC_ID,
-        SUBTOPIC_TOPIC_ID_1
+        FRACTIONS_SUBTOPIC_TOPIC_ID_1,
+        FRACTIONS_SUBTOPIC_LIST_SIZE
       )
     ).use {
       testCoroutineDispatchers.runCurrent()
-
-      onView(withId(R.id.previous_navigation_card))
-        .check(matches(isDisplayed()))
-
-      onView(withId(R.id.next_navigation_card))
-        .check(matches(isDisplayed()))
+      onView(withId(R.id.navigation_card_container)).perform(scrollTo())
+      onView(withId(R.id.previous_navigation_card)).check(matches(isDisplayed()))
+      onView(withId(R.id.next_navigation_card)).check(matches(isDisplayed()))
     }
   }
 
+//  TODO(#4631): Remove this once #4235 is resolved.
+  @DisableAccessibilityChecks
   @Test
-  fun testRevisionCardTestActivity_fractionSubtopicId3_checkNextNavCardIsNotDisplayed() {
+  fun testRevisionCardTestActivity_fractionSubtopicId3_checkOnlyNextNavCardIsNotDisplayed() {
     launch<RevisionCardActivity>(
       createRevisionCardActivityIntent(
         context,
         profileId.internalId,
         FRACTIONS_TOPIC_ID,
-        SUBTOPIC_TOPIC_ID_3
+        FRACTIONS_SUBTOPIC_TOPIC_ID_3,
+        FRACTIONS_SUBTOPIC_LIST_SIZE
       )
     ).use {
       testCoroutineDispatchers.runCurrent()
-
-      onView(withId(R.id.next_navigation_card))
-        .check(matches(not(isDisplayed())))
+      onView(withId(R.id.navigation_card_container)).perform(scrollTo())
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.previous_navigation_card)).check(matches(isDisplayed()))
+      onView(withId(R.id.next_navigation_card)).check(matches(not(isDisplayed())))
     }
   }
 
+//  TODO(#4631): Remove this once #4235 is resolved.
+  @DisableAccessibilityChecks
   @Test
   fun testRevisionCardTestActivity_fracSubtopicId1_clickPrevNavCard_opensRevisionCardActivity() {
     launch<RevisionCardActivity>(
       createRevisionCardActivityIntent(
         context,
         profileId.internalId,
         FRACTIONS_TOPIC_ID,
-        SUBTOPIC_TOPIC_ID_1
+        FRACTIONS_SUBTOPIC_TOPIC_ID_1,
+        FRACTIONS_SUBTOPIC_LIST_SIZE
       )
     ).use {
       testCoroutineDispatchers.runCurrent()
-
       onView(withId(R.id.previous_navigation_card)).perform(click())
       testCoroutineDispatchers.runCurrent()
       intended(hasComponent(RevisionCardActivity::class.java.name))
-      intended(hasExtra(SUBTOPIC_ID_EXTRA_KEY, SUBTOPIC_TOPIC_ID_1 - 1))
+      intended(hasExtra(SUBTOPIC_ID_EXTRA_KEY, FRACTIONS_SUBTOPIC_TOPIC_ID_0))
     }
   }
 
+//  TODO(#4631): Remove this once #4235 is resolved.
```

Comment:
Please fix spacing for this comment. Ditto elsewhere in this test.

Reply:
done